inc/Engine/Cache/Purge.php

```diff
@@ -246,4 +256,50 @@ private function get_post_terms_urls( WP_Post $post ) {
 		*/
 		return apply_filters( 'rocket_post_terms_urls', $urls );
 	}
+
+	/**
+	 * Purge single cache file(s) added in the Never Cache URL(s).
+	 *
+	 * @param array $old_value An array of previous values for the settings.
+	 * @param array $value An array of submitted values for the settings.
+	 * @return void
+	 */
+	public function purge_cache_reject_uri_partially( $old_value, $value ): void {
```

Comment:
In the subscriber you typecast the parameters, and here you don't, both should be the same for consistency